api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,14 +117,23 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
-	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
+	<changeSet author="Kelechi+iheanyi" id="20180508-1000">
 		<preConditions onFail="MARK_RAN">
-			<not><tableExists tableName="condition"/></not>
+			<tableExists tableName="conditions"/>
 		</preConditions>
 		<comment>
-			Add condition table
+			Rename conditions table to emrapi_conditions
```

Comment:
Why are you renaming the table, I thought you should be updating it to match the new design

Reply:
@wluyima 
We are renaming the table because of some edge cases we saw in the implementation. 
Assume we are running a fresh instance of the core platform and the emrapi module.
If we update the `conditions` table here, the emrapi module tries to create a new `conditions` table which throws errors.

However, the following implementation handles that edge case successfully. On the core platform, the conditions table is renamed to emrapi_conditions. Then a conditions table is created and data migrated to it. On the emrapi module, the emrapi_conditions table is used.

Reply:
You should not be renaming the table, in any case it's no longer going to live in the emrapi module so there if no need to add that prefix

Reply:
@wluyima  , wat do u mean if the table exists , just do the migration ..? i think that will be just do the modification.  because migration is posiible only after first renaming it.